dbldatagen/data_generator.py

```diff
@@ -1604,3 +1617,51 @@ def scriptMerge(self, tgtName=None, srcName=None, updateExpr=None, delExpr=None,
             result = HtmlUtils.formatCodeAsHtml(results)
 
         return result
+
+    @staticmethod
+    def fromDict(options):
```

Comment:
Make sure to have explicit tests for this covering the following use cases:

1 - with simple options
2 - with composite (object valued options) 

See the examples on the following page for object valued options - i.e DateRange, Distribution objects